README.md

```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
+Die Anmeldung ist ziemlich so, wie oben schon beschrieben. Hier werden nur die Änderungen erläutert (gesamten Ablauf bitte oben nachlesen).
+
+ * Sensor ID:\
+ Hier den hinteren Teil der DEVEUI eingeben (wenn also - siehe oben - die DEVEUI *00 D0 C0 00 C3 19 7C E8* heißt, sind die hinteren 4 Byte ( also *C3197CE8*) in dezimal umgerechnet ( also hier 3273227496 ) einzugeben.
+ * Sensor Board:\
+ Hier **TTN** auswählen (über die kleinen Pfeile rechts).
+ 
+
+### HTTP-Integration
+Um die Daten von TTN an *sensor.community* zu bekommen, muss noch die HTTP-Integration bei TTN eingestellt werden.\
+In der TTN-Console *Applications* anklicken und dann die Applikation des Geigerzählers (z.B. *geiger_20200205*) anklicken. Dann rechts oben in der Leiste mit *Overview*, *Devices*, *Payload Formats*, *Integrations*, *Data*, *Settings* das **Integrations** anklicken. Hier dann über **add integration** die **HTTP Integration** auswählen.
+
+Nun die angezeigten Felder ausfüllen:\
+ **Process ID:**\
+ 	Hier einen beliebigen Namen für diese Integration vergeben\
+ **Access Key:**\
+ 	Hier einmal rein klicken und den *default key* auswählen\
+ **URL:**\
+ 	Hier nun die URL zum ttn2luft-Programm eingeben: <https://ttn2luft.citysensor.de>\
+ **Method:**\
+ 	da steht schon *POST*, das lassen\
+ **Authorization:**\
+ 	bleibt leer\
+ **Custom Header Name:**\
+ 	hier kommt der Text **X-SSID** rein\
+ **Custom Header value:**\
+ 	und da dann die SSID des Sensors (also die Nummer, die man bei der Anmeldung bei sensor.community erhalten hat, *NICHT* die Chip-ID).
+
+Dann rechts unten auf **Add integration** klicken. Das war's dann.\
+Und so etwa sieht das dann ausgefüllt aus:\
+!["HTTP-Integration filled"](images/http_integration_filled.png)
+
+### TTN-Payload (Beispiel)
+Um in der TTN-Console nicht nur die Datenbytes sondern lesbare Werte zu sehen, kann ein kleines Script als payload-Decoder eingefügt werden.\
+Auf der *The Things Network* Webseite einloggen, über **Applications** die oben angelegte Application suchen und dann anklicken. Hier in der Menü-Zeile den Tab **Payload Formats** auswählen. Dann den folgenden Code in das Feld einfügen (den vorhandenen Code überschreiben):
+
+```
+function Decoder(bytes, port) {
+  // Decode an uplink message from a buffer
+  // (array) of bytes to an object of fields.
+  var decoded = {};
+  if(port == 1) {
+  decoded.counts = ((bytes[0]*256 + bytes[1]) * 256 + bytes[2]) * 256  + bytes[3];
+  decoded.sample_time = (bytes[4] * 256 + bytes[5]) * 256 + bytes[6];
+  decoded.tube = bytes[9];
+  var minor = (bytes[7]&0xF)+(bytes[8]>>4) ;
+  decoded.sw_version="" + (bytes[7]>>4) + "." + minor + "." + (bytes[8]&0xF);
+  }
+  if (port === 2) {
+    decoded.temp = ((bytes[0] * 256 + bytes[1]) / 10) + "°C";
+    decoded.humi = bytes[2] / 2 + "%";
+    decoded.press = ((bytes[3] * 256 + bytes[4]) / 10) + "hPa";
+  }
+  return decoded;
+}
+```
```

Comment:
Overall, this README is way too long and it's getting worse (longer) all the time.

There is an open ticket about writing real docs: #52 and to solve the language issue #168 - this should be done soon.

Reply:
I know. But I don't want to translate all these stuff to english. If someone else is willig to do this, please do it.

Reply:
if the language ticket is solved first, you can write/update in german.